job-dsls/jobs/kie/main/jbake_web_publishing.groovy

```diff
@@ -0,0 +1,86 @@
+/**
+ * job that publishes automatically the ${repo}-website
+ */
+
+import org.kie.jenkins.jobdsl.Constants
+
+def baseBranch=Constants.BRANCH
+// creation of folder
+folder("KIE")
+folder("KIE/${baseBranch}")
+folder("KIE/${baseBranch}/webs")
+def folderPath="KIE/${baseBranch}/webs"
+
+def final DEFAULTS = [
+        repository : "drools-website",
+]
+
+def final REPO_CONFIGS = [
+        "drools"      : [],
```

Comment:
why this empty repo configuration?

Reply:
because REPO_CONFIGS inherits from DEFAULTS, and here the repository is "drools-website". The later coming loop iterates over REPO_CONFIG, so drools must be inside. 
Maybe I am wrong and you could do this on more nice programmer style?

Reply:
Indeed. @mareknovotny see #L29-L31
```
    Closure<Object> get = { String key -> reps.value[key] ?: DEFAULTS[key] }

    String REPO = get("repository")
```
the values are taken from `DEFAULTS` in case it does not exist for specific repo.